src/main/java/javax/resource/cci/ResourceWarning.java

```diff
@@ -98,7 +98,7 @@ public ResourceWarning getLinkedWarning() {
      *
      * @param warning <code>ResourceWarning</code> to be added to the chain.
      *
-     * @deprecated J2SE release 1.4 supports a chained exception facility 
+     * @deprecated Java release 1.4 supports a chained exception facility 
```

Comment:
The updates are inconsistent between "Java release 1.4" and "Java SE release 1.4" in several places

Reply:
Put them all back to J2SE as advised by @bshannon